docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+that tag since the last value for each series is what gets defined if you're not filtering by time range or other 
+criteria.
+
+```
+@my_data.h2o_temperature{state}
+```
+
+Returns these results:
+
+| state |
+| --- |
+| CA |
+| CA |
+| WA |
+
+#### Filtering on Criteria
+You can limit the results returned in the query by specifying filtering criteria in the *Column Filter Predicate* block. 
+Here's an example that filters based on the location:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, state == "CA"}
+```
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+Note that `state` is returned in the result because it was listed in the filter criteria. You can have more complex 
+logic to match criteria like:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, location == "coyote_creek" OR location == "puget_sound"}
+```
+
+See that while the column names are separated by commas, the filter criteria is separated by the different matching 
+clauses.
+
+
+#### Matching Critera
+There are many operators you can use to filter down data. Here is the list of operators:
+* `==` (equal to)
+* `!=` (not equal)
+* `=~` (regex match)
+* `!~` (not a regex match)
+* `>` (greater than)
+* `>=` (greater than or equal to)
+* `<` (less than)
+* `<=` (less than or equal to)
+* `in`
+
+The regex and in operators are probably the trickiest ones to work with, so here are a few examples:
+
+```
+// this is a comment because it has // before it
+
+// status is a tag and this will match any stats between 200 and 500
+@my_data.http_requests{status =~ /[2-5][0-9][0-9]/}
+
+// will match states in the array
+@my_data.h2o_temperature{state in ["CA", "WA"]}
+```
+
+### Aggregate, Select, Transform and Group Data
+Flux has many built in functions to work with time series data. **Aggregate** functions refer to functions that summarize 
+some period of time. They take N rows and output a single result, examples include count, mean, and percentile, 
+among many others. **Select** functions select a set of rows from an input set based on some criteria. Min, max, top, and 
+bottom are examples of selectors. Some selectors also double as aggregates like min and max.
+
+This is an important distinction. An aggregate produces new data from some source data. That new data is frequently a 
+summary statistic that describes the source data in some meaningful way. A selector specifies criteria for selecting 
+rows from some source data. So you're not transforming the data when using a selector. For example, `count` is not a 
+function that could be used as a selector.
+
+#### Sample Dataset
+Here's an example dataset that we'll use for the rest of the examples in this section. The raw number timestamps 
+have been replaced with human readable times and the values have been set to simple whole numbers to make things 
```

Comment:
IMO, Just use single digit timestamps like 1, 2, 3, 4 here. They would be easier to understand than the "human readable times".